charts/dlb-device-plugin/values.yaml

```diff
@@ -0,0 +1,16 @@
+name: dlbdeviceplugin-sample
+
+image:
+  hub: intel  
+  tag: ""
+
+initImage:
+  hub: intel  
+  tag: ""
+
+logLevel: 4
+
+nodeSelector:
+  intel.feature.node.kubernetes.io/dlb: 'true'
+
+nodeFeatureRule: true
```

Comment:
the existing plugins moved to `false` so better keep these aligned

Reply:
do you mean nodeFeatureRules set to false?